src/App.js

```diff
@@ -23,6 +23,14 @@ function App() {
     setHexBuffer(buffer);
   };
 
+  const fileChange = async e => {
+    const file = (e.target.files || [])[0];
+    if (file) {
+      const b = new Uint8Array(await file.arrayBuffer());
+      setHex([...b].map(c => c.toString(16).padStart(2, '0')).join(' '));
```

Comment:
What's the reason for adding '00' at the beginning?

Reply:
I just thought it looked nicer like how it would be entered, manually. I am dyslexic and I can read hex easier if it's 0-padded (so every byte is the same number of characters.) It doesn't just add `00` but pads to 2-digit hex, so `0` ends up `00` and `1` ends up `01` but `ff` is still `ff`.

Reply:
I see. Can you extract this ArrayBuffer => hex method to a util function in a different file and add a test for it?

Reply:
I think this is just the same as `bufferToPrettyHex` but is more efficient (using map/join and padStart, instead of for-loop and if-statements.) I didn't see that function, and probably would have used it, on closer inspection, even though I think it is a lil clunky, comparatively. 

If I replace the function like this, all your tests pass:

```js
export const bufferToPrettyHex = b => [...b].map(c => c.toString(16).padStart(2, '0')).join(' ');
```

I pushed this change, so it should have all the same tests you wrote, and a more efficient `bufferToPrettyHex`.